docs/wiki/deployment/log-aggregation.md

```diff
@@ -74,19 +68,17 @@ sourcetype = osquery_warning
 
 ### Rsyslog
 
-[rsyslog](http://www.rsyslog.com/) is a tried and testing unix log forwarding service. If you're deploying osqueryd in a production Linux environment where you don't have to worry about lossy network connections, this may be your best option.
+[rsyslog](http://www.rsyslog.com/) is a tried and testing UNIX log forwarding service. If you are deploying **osqueryd** in a production Linux environment where you do not have to worry about lossy network connections, this may be your best option.
 
 ## Analyzing logs
 
-The way in which you analyze logs is very dependent on how you're aggregating logs. At the end of the day, osquery produces results logs in JSON format, so the logs are very easy to analyze on most modern backend log aggregation platforms.
+The way in which you analyze logs is very dependent on you aggregate logs. At the end of the day, osquery produces results logs in JSON format, so the logs are very easy to analyze on most modern backend log aggregation platforms.
```

Comment:
nit: Should be `on how you aggregate logs` or something like: `... very depended on how logs are aggregated`.